assets/js/types/type-defs/addresses.ts

```diff
@@ -1,13 +1,13 @@
-export interface Address {
+export interface EnteredAddress {
+	first_name: string;
+	last_name: string;
+	company?: string;
 	address_1: string;
 	address_2?: string;
-	city?: string;
-	company?: string;
+	city: string;
+	state: string;
 	country: string;
-	email: string;
-	first_name: string;
-	last_name: string;
-	phone?: string;
 	postcode: string;
-	state?: string;
+	phone?: string;
+	email?: string;
```

Comment:
I'm wondering about whether it is correct to use this type for the shipping address, email is not a field on a shipping address, but here it is marked as optional. So if we're writing code, our IDE will tell us `shippingAddress.email` is fine to do, but it actually isn't. I think we should still try to split them up into distinct address types if possible.